src/libawkward/array/RecordArray.cpp

```diff
@@ -352,7 +352,7 @@ namespace awkward {
   }
 
   const std::shared_ptr<Content> RecordArray::flatten(int64_t axis) const {
-    throw std::runtime_error("FIXME: not implemented");
+    return getitem_range_nowrap(0, length() - 1);
```

Comment:
Actually, this line does nothing: a range from `0` to `length() - 1` is the whole array; you get the whole thing back.

Like `Record`, flattening a `RecordArray` should raise an error.